packages/build/src/steps/error.js

```diff
@@ -33,7 +33,12 @@ export const handleStepError = function ({
   }
 
   const fullErrorInfo = getFullErrorInfo({ error: newError, colors: false, debug })
-  const { type } = fullErrorInfo
+  const {
+    errorInfo: { location: { packageName } = {} },
+    message,
+    title,
+    type,
+  } = fullErrorInfo
```

Comment:
For reference I double checked and seems like the `errorInfo` is always populated:
- https://github.com/netlify/build/blob/12ebc86ab192312b02ed8b832ac66be452194b3f/packages/build/src/error/parse/parse.js#L14-L27C14